rmd - readme - hw01.Rmd

```diff
@@ -0,0 +1,33 @@
+---
+title: "Bio-hw01"
+author: "Frank"
+date: "4/8/2020"
+output: html_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+# My personal introduction
+
+## Brief Introduction
+
+My name is Feiyang **Frank** Yu. I am from MAPSS program, majored in political science and sociology. I am from the southern part of China. I have try to learn *Python* by myself two years ago, and this is the first time I learn something about R. It seems to me that the use of R is like using the functions in Python. It is efficient for data scientists, which is very helpful to me as well.
```

Comment:
Good use of bold and italics.  Just two typos: "tried" instead of "try" (line 17) and "several" instead of "severl" (line 22).  Otherwise, markdown looks fine, although it went beyond the scope of the assignment!